src/components/AudioParameter.vue

```diff
@@ -100,12 +92,48 @@ export default defineComponent({
       }
     };
 
+    const getClipPath = (): string => {
+      if (!props.clip) {
+        return "";
+      } else {
+        if (props.type == "vowel") {
+          return "clip-path: inset(-50% -50% -50% 50%)";
+        } else {
+          return "clip-path: inset(-50% 50% -50% -50%)";
+        }
+      }
+    };

```

Comment:
リアクティブにしないとバグるかもしれません。
引数がない`get`は、すべて`computed`に置き換えられると思います。
It may be buggy if you don't keep this reactive.
I think any `get` with no arguments will be replaced by `computed`.